### ISSUE TITLE
MultiPoolMiner

### ISSUE THREAD

**Issue by tooviral** [boredazfcuk/mining#2]
any way to get this running for multipoolminer?

**Comment by boredazfcuk** [comment]
Not easily. MultiPoolMiner utilises batch and powershell scripts to manage everything. I currently check to see if the executables (NiceHash.exe and AwesomeMiner.exe) are open and then kill them and the miner programs before relaunching. With MPM though, only the cmd.exe and pwsh.exe processes will be running and killing these off indiscriminately could cause system issues... Identifying which of the 50+ MPM scripts are currently running and identifying the parent executable will take a good deal of reworking.

I had switched to MPM but I've found it earns less than AwesomeMiner so I've switched back.

To be honest, it's probably a better idea to re-write Monitor-AwesomeMiner script in powershell, and incorporate it into the MPM project.
